### PR TITLE
[1.24.0] Docs for kots remove command

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/kots-cli/remove.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+linktitle: "remove"
+title: kots remove
+weight: 90120
+---
+
+Remove application reference from Admin Console.
+Multiple applications can be installed using the `Add a new application` button in Admin Console or using the `kots install` command.
+All installed applications will appear in Admin Console.
+The `kots remove` command can be used to remove them.
+Note that deployed application will not be removed from the cluster, only it's reference will be removed from Admin Console.
```
Comment (markpundsack) [inline content/kots-cli/remove.md:12]:
```suggestion
Note that the deployed application will not be removed from the cluster; only it's reference will be removed from Admin Console.
```

File: content/kots-cli/remove.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Usage
+```bash
+kubectl kots remove [app-slug]
+```
+* _Replace `[app-slug]` with the slug for the KOTS application that should be removed (required)._
```
Comment (markpundsack) [inline content/kots-cli/remove.md:18]: You usually don't need to say "replace". People know how CLI docs work, right? Just say "app-slug is..."

File: content/kots-cli/remove.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* _If the KOTS application has been packaged by Replicated Vendor, the `--license-file` flag must be provided._
```
Comment (markpundsack) [inline content/kots-cli/remove.md:19]: Why is the license file required on remove? Is that just for a safety check? Seems burdensome. Also, how would a Kots application not be packaged by Replicated Vendor? Why is Vendor capitalized? Needs a "the".

File: content/kots-cli/remove.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* _Provide `[flags]` according to the table below_
+
+This command supports all [global flags](/kots-cli/global-flags/) and also:
+
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--force` |  bool  |  if application was deployed, the command will fail unless the force flag is specified |
```
Comment (markpundsack) [inline content/kots-cli/remove.md:27]:
```suggestion
| `--force` |  bool  |  Remove the reference even if the application has already been deployed. |
```

File: content/kots-cli/remove.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+### Example
+```bash
+kubectl kots remove sentry -n default
```
Comment (markpundsack) [inline content/kots-cli/remove.md:31]: The text above says a license is required for nearly all instances, so maybe it should be provided here? (or drop that requirement)

Comment (divolgin) [reply content/kots-cli/remove.md:30]: that was just copy/paste stuff from another doc

**Pull request merged**